src/public/mathfield-element.ts

```diff
@@ -1036,16 +1026,26 @@ export class MathfieldElement extends HTMLElement implements Mathfield {
     // This.setAttribute('aria-multiline', 'false');
     if (!this.hasAttribute('tabindex')) this.setAttribute('tabindex', '0');
 
+    const slot =
```

Comment:
Why did you have the change the order?

Reply:
OK, I understand why now. I think it's OK, but please test with both Safari and Firefox.